localstack-core/localstack/aws/handlers/logging.py

```diff
@@ -21,16 +24,24 @@ class ExceptionLogger(ExceptionHandler):
     def __init__(self, logger=None):
         self.logger = logger or LOG
 
+        # Moto may not be available in stripped-down versions of LocalStack, like LocalStack S3 image.
+        self._moto_service_exception = types.EllipsisType
+        with contextlib.suppress(ModuleNotFoundError, AttributeError):
+            self._moto_service_exception = importlib.import_module(
+                "moto.core.exceptions"
+            ).ServiceException
```

Comment:
nit: i think i would prefer this, simply because the dynamic import via `import_module` makes it harder/impossible for an IDE to detect the import, whereas `from moto.core.exceptions import ServiceException` will show up in searches, module uses, etc.

```python
        try:
            from moto.core.exceptions import ServiceException
            self._moto_service_exception = ServiceException
        except (ModuleNotFoundError, AttributeError):
            self._moto_service_exception = types.EllipsisType
```